promscale/installation/docker.md

```diff
@@ -100,9 +101,15 @@ When you are succecssdfully receiving metrics from your Promscale instalaltion,
 </procedure>
 
 ## Install Promscale with a Docker compose file
-You can use a Docker compose file to create containers for TimescaleDB, Promscale, and the `node_exporter`, using a single command. To help you get started, we have created a sample Docker compose file, available from the [Promscale GitHub repository][promscale-docker-compose].
+You can use a Docker compose file to create containers for TimescaleDB,
+Promscale, and the `node_exporter`, using a single command. To help you get
+started, we have created a sample Docker compose file, available from the
+[Promscale GitHub repository][promscale-docker-compose].
 
-Before you begin, make sure that you have the `prometheus.yml` configuration file in the same directory as `docker-compose.yml`.
+Before you begin, make sure that you have your `prometheus.yml` configuration
+file is in the same directory as `docker-compose.yml`. For more information
```

Comment:
```suggestion
file in the same directory as `docker-compose.yml`. For more information
```

Reply:
This content was removed so the problem doesn't exist anymore